src/main/java/edu/kis/powp/jobs2d/drivers/DriverComposite.java

```diff
@@ -0,0 +1,51 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.Visitor;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DriverComposite implements DriverCommand {
```

Comment:
Sorry, but this is not a DriverComposite (composite in Job2dDriver hierarchy) but DriverCommandComposite (which is not a topic of this task).